scripts/run_docker_build.sh

```diff
@@ -18,14 +18,19 @@ CONDARC
 )
 
 cat << EOF | docker run -i \
-                        -v ${REPO_ROOT}/recipes:/conda-recipes \
+	                -v ${REPO_ROOT}:/staged-recipes \
```

Comment:
Amazingly missed that there was a whitespace error here. Fixing in PR ( https://github.com/conda-forge/staged-recipes/pull/1408 ).